charts/elasticsearch-data/values.yaml

```diff
@@ -0,0 +1,179 @@
+  # -- Elasticsearch master node role
+  node.master: "false"
+  # -- Elasticsearch ingest node role
+  node.ingest: "true"
+  # -- Elasticsearch data node role
+  node.data: "true"
+  # -- Elasticsearch ml node role
+  node.ml: "false"
+  # -- Elasticsearch JVM options
+  ES_JAVA_OPTS: "-Xms2048m -Xmx2048m"
+  # -- Elasticsearch enable compression between nodes
+  transport.tcp.compress: "true"
+  # -- Elasticsearch enable memory lock to avoid swapping
+  bootstrap.memory_lock: "true"
+  # -- Elasticsearch node attribute type
+  node.attr.type: search
+  # -- Elasticsearch GCE discovery log level
+  logger.org.elasticsearch.discovery.gce: TRACE
+
+serviceAccount:
+  # -- Specifies whether a service account should be created
+  create: true
+  # -- Annotations to add to the service account
+  annotations: {}
+  # -- The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+prometheus:
+  # -- Deploy a ServiceMonitor for Prometheus scrapping
+  enabled: false
+  # -- Annotations to include in the ServiceMonitor
+  annotations: 
+    release: prometheus
+    app: prometheus-operator
+  exporter:
+  # -- Exporter image to deploy as a sidecar container
+    image: justwatch/elasticsearch_exporter:1.1.0
+
+# -- Configurable annotations applied to all Elasticsearch pods
+podAnnotations: {}
+
+# -- Allows you to set the securityContext for the pod
+podSecurityContext: {}
+  # fsGroup: 2000
+
+# -- Allows you to set the securityContext for the container
+securityContext:
+  capabilities:
+    add:
+    - IPC_LOCK
+    - SYS_RESOURCE
+
+service:
+  # -- Kubernetes service type
+  type: ClusterIP
+  # -- Kubernetes service port, used by Ingress to expose Elasticsearch pods
+  port: 9200
+
+ingress:
+  # -- Enable Kubernetes Ingress to expose Elasticsearch pods
+  enabled: false
+  # -- Annotations for Kubernetes Ingress
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  
+  # -- Host and path for Kubernetes Ingress. See values.yaml for an example
+  hosts: []
+  #  - host: chart-example.local
+  #    paths: /
+  
+  # -- TLS secret for exposing Elasticsearch with https. See values.yaml for an example
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+    # -- CPU limits for the StatefulSet
+    cpu: 1000m
+    # -- Memory limits for the StatefulSet
+    memory: 3Gi
+  requests:
+    # -- CPU requests for the StatefulSet
+    cpu: 100m
```

Comment:
I think this should be 1000m

Reply:
Changed, thnx! I'll need your approval again in case you do not see anything else ::